metricflow/test/model/validations/test_validity_param_definitions.py

```diff
@@ -70,6 +70,7 @@ def test_validity_window_must_have_a_start() -> None:
     validity_window_file = YamlConfigFile(filepath="inline_for_test", contents=yaml_contents)
     model = parse_yaml_files_to_validation_ready_model([base_model_file(), validity_window_file])
 
+    #foo bar test commit
```

Comment:
![](https://media.giphy.com/media/APKXp0zzM7k3JFOZsW/giphy.gif)

🤪 

(For real though, looking forward to the update or new PR with the proper changes.)